src/components/Dropdown/Menu/Item/Item.tsx

```diff
@@ -1,63 +1,64 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import classNames from 'classnames';
 
 import styles from './Item.module.css';
 
-export interface ItemProps {
-  className?: string;
+export type ButtonElementProps = React.ButtonHTMLAttributes<HTMLButtonElement>;
+
+type AnchorElementProps = React.AnchorHTMLAttributes<HTMLAnchorElement> & {
```

Comment:
Just curious: Anchor elements should already have `href` - did it need to be added here?